fcrepo-http-api/pom.xml

```diff
@@ -77,11 +77,6 @@
       <version>${project.version}</version>
     </dependency>
 
-    <dependency>
-      <groupId>org.fcrepo</groupId>
-      <artifactId>fcrepo-connector-file</artifactId>
-      <version>${project.version}</version>
```

Comment:
It seems like moving the projection tests into `fcrepo-webapp` might make the most sense.